### PR TITLE
simplify and finish unbond example

### LINKED ISSUE #9782: unbondExample.contract.js is incomplete
## What is the Problem Being Solved?

[unbondExample.contract.js](https://github.com/Agoric/agoric-sdk/blob/8a8fed33ee6e578227775ed5ad9297f7336fb4be/packages/orchestration/src/examples/unbondExample.contract.js) currently has a lot of TODOs and incomplete functionality, but is documented on our docs site: https://docs.agoric.com/guides/orchestration/getting-started/contract-walkthrough/cross-chain-unbond.html

```[tasklist]
### Tasks
- [ ] https://github.com/Agoric/agoric-sdk/issues/9796
- [ ] https://github.com/Agoric/agoric-sdk/issues/10016
- [ ] https://github.com/Agoric/agoric-sdk/issues/10017
```

### PR DESCRIPTION
**Pull request by turadg**
closes: #9782

## Description

#9782 was to finish the example of "unbond and liquidate". We've punted Liquid Staking,
- https://github.com/Agoric/agoric-sdk/issues/10017

So to get a complete working example this does a transfer after unbonding.

### Security Considerations
Not being deployed

### Scaling Considerations
Not being deployed

### Documentation Considerations
May become part of documentation, up to DevRel

### Testing Considerations
CI

### Upgrade Considerations
Not being deployed

### CODE REVIEW

File: packages/orchestration/src/examples/unbond.flows.js [hunk 1]
```diff
@@ -30,11 +30,7 @@ export const unbondAndTransfer = async (orch, { zcf }) => {
   const stride = await orch.getChain('stride');
   const strideAccount = await stride.makeAccount();
 
-  // TODO the `TIA` string actually needs to be the Brand from AgoricNames
-  // const tiaAmt = await celestiaAccount.getBalance('TIA');
-  // await celestiaAccount.transfer(tiaAmt, strideAccount.getAddress());
-  // TODO https://github.com/Agoric/agoric-sdk/issues/10017
-  // await strideAccount.liquidStake(tiaAmt);
-  console.log(osmoAccount, strideAccount);
+  const balance = await osmoAccount.getBalance('OSMO');
```
Comment (0xpatrickdev) [inline packages/orchestration/src/examples/unbond.flows.js]: `OSMO` works, and it doesn't need to be `uosmo`? Maybe grab `stakingTokens[0]` from chainInfo?

**Review by 0xpatrickdev — COMMENTED** [review]
> L20 // In a real world scenario, accounts would be re-used across invocations of the handler

Since #9796 is closed, and this PR will close 9782, can I request this is written as a continuing invitation? This would allow us to test in `multichain-testing` and improve the fidelity of our example contracts

**Comment by 0xpatrickdev** [comment]
> > L20 // In a real world scenario, accounts would be re-used across invocations of the handler
> 
> Since #9796 is closed, and this PR will close 9782, can I request this is written as a continuing invitation? This would allow us to test in `multichain-testing` and improve the fidelity of our example contracts

Settled on a comment that points the reader to staking-combinations for an example of what I was asking for

**Review by 0xpatrickdev — APPROVED** [review]

**Comment by turadg** [comment]
@mergify requeue

**Comment by dckc** [comment]
oh crud... I'm too slow to note the examples README should be updated:

https://github.com/Agoric/agoric-sdk/blob/741bbd1caa576b7795d5bde517ca473b716c35db/packages/orchestration/src/examples/README.md?plain=1#L9-L14

**Pull request merged**